scheduler-server/sql/scheduler/tables/scheduling_specification_goals.sql

```diff
@@ -3,10 +3,12 @@ create table scheduling_specification_goals (
   goal_id integer not null,
   priority integer
     not null
-    default null -- Nulls are detected and replaced with the next
+    default null, -- Nulls are detected and replaced with the next
                  -- available priority by the insert trigger
-    constraint non_negative_specification_goal_priority check (priority >= 0),
+  enabled boolean default true,
 
+  constraint non_negative_specification_goal_priority
```

Comment:
Is there a semantic difference between attaching a constraint to a column, vs having the constraint at the top level of the table? Or is that merely a matter of taste?